packages/popover/components/PopoverListItem.tsx

```diff
@@ -80,11 +80,10 @@ const PopoverListItem = (props: PopoverListItemProps) => {
           itemContent: children
         };
       }
-      const isItemGraphic = (
-        child
-      ): child is typeof PopoverListItemIcon | typeof PopoverListItemAvatar =>
-        (React.isValidElement(child) && child.type === PopoverListItemIcon) ||
-        child.type === PopoverListItemAvatar;
+      const isItemGraphic = (child): boolean =>
```

Comment:
This took me longer to fix than I would like to admit. The problem was that here we are basically turning the child into a type that did not have props on it. Since a react element at run time has props and this functional component does not, it does not work. An even better idea would be to refactor this to explicitly take an element for the start and end position, but that seemed out of scope.